src/components/shared-components/navbar/NavBar.jsx

```diff
@@ -38,25 +40,25 @@ const NavBar = () => {
       <Navbar.Collapse id='responsive-navbar-nav'>
         <Nav className='ms-auto' defaultActiveKey='#home'>
           <Nav.Item>
-            <Nav.Link as={Link} to='/case-studies' onClick={() => updateExpanded(false)}>
+            <Nav.Link as={Link} to='/case-studies' active={location.pathname === '/case-studies'} onClick={() => updateExpanded(false)}>
```

Comment:
Yes, NavLink does have the isActive property. Still, when I tried using NavLink instead of Link the CSS broke in certain parts. Implementing "useLocation" from react-router-dom and then comparing the current location to the page name seemed like less of a hassle when it comes to implementation, it was faster and easier than changing the CSS. Practically more or less the same amount of lines of code.